ios/Plugin/PrivacyScreenPlugin.swift

```diff
@@ -50,6 +66,37 @@ public class PrivacyScreenPlugin: CAPPlugin {
         }
     }
 
+    @objc private func didDetectRecording() {
+        guard self.isEnabled else {
+            return
+        }
+
+        if #available(iOS 11.0, *) {
+            if UIScreen.main.isCaptured {
+                DispatchQueue.main.async {
+                    self.bridge?.viewController?.present(self.privacyViewController!, animated: false, completion: nil)
+                }
+            } else {
+                self.privacyViewController?.dismiss(animated: false, completion: nil)
+            }
+        }
+
+    }
+
+    @objc private func didDetectScreenshot() {
+        guard self.isEnabled else {
+            return
+        }
+
+        self.alert = UIAlertController(title: "Attention",
+                                              message: "Screenshots may contain sensitive data. Do not keep them on your phone!",
+                                              preferredStyle: .alert)
+        self.alert?.addAction(UIAlertAction(title: "OK", style: .default))
+        DispatchQueue.main.async {
+            self.bridge?.viewController?.present(self.alert!, animated: false, completion: nil)
+        }
```

Comment:
The plugin should not interact with the user. 
I propose an event listener instead.
Example: https://github.com/ionic-team/capacitor-plugins/tree/main/app#addlistenerappstatechange-

Everyone should decide on their own how to deal with this event.
For example, an app wants to display the message in a different language.

Reply:
Sure. Also i wanted to pass these strings in configs. where user could define if this alert should appear.